docker-compose.yaml

```diff
@@ -14,35 +14,5 @@ services:
     environment:
       ROS_DOMAIN_ID: ${ROS_DOMAIN_ID}
       RMW_IMPLEMENTATION: ${RMW_IMPLEMENTATION}
-    depends_on:
-      - formant-agent
```

Comment:
@Infinite-Echo  do we need to get rid of this formant-agent piece? our current setup is to run the formant agent in a docker launched from here.

Reply:
@Infinite-Echo  added back so i can close in this sprint :)